internal/manifests/collector/container.go

```diff
@@ -107,6 +107,15 @@ func Container(cfg config.Config, logger logr.Logger, otelcol v1alpha1.OpenTelem
 		},
 	})
 
+	if len(otelcol.Spec.ConfigMaps) > 0 {
+		for keyCfgMap := range otelcol.Spec.ConfigMaps {
+			volumeMounts = append(volumeMounts, corev1.VolumeMount{
+				Name:      otelcol.Spec.ConfigMaps[keyCfgMap].Name,
+				MountPath: otelcol.Spec.ConfigMaps[keyCfgMap].MountPath + "/" + otelcol.Spec.ConfigMaps[keyCfgMap].Name,
```

Comment:
Do we need to append the configmap name to the mountpath?

Reply:
The user specifies this explicitly, so I would say no. It's up to them to do the right thing here.

Reply:
Just removed the Name's append; I also think it makes sense thinking about the user perspective.

Reply:
Don't we not want the configmap name here?